### DISCUSSION TITLE
Plan For Customer Demo

### DISCUSSION THREAD

**Discussion by Thandavas** [thinkarlabs/P23-001-RehabTracker#18] [General]
[Milestone 1]- [Estimated date : 08-03-2023]
1.Therapist Login with email and password. On successful login he will land on sessions page.
2. In Sessions screen for selected patient Therapist can start new session
3. New-Session will redirect to Exercise page In that he can select  Shoulder , Squats , Knee Lift  Exercise as of now 
4. After Selecting the Exercise it will redirect to Track Page where Therapist can Observe , Track , Record the patient exercise
5.  In Sessions screen Therapist can see completed sessions recording -- static

[Milestone 2]-[Estimated date : 31-03-2023]
1.patients session screen Implementation
2.Therapist can add , edit , delete patient details
3. Therapist can add , edit ,delete Injury Type Details 
4. All the patients Exercise performance details will captured while patient doing exercise

**Comment by srivatsapraveen** [comment]
I think this is a good description for the demo script. When are we targeting Milestone 1 and 2? (Both by mid-march?)
